src/basic/DAXPY-Hip.cpp

```diff
@@ -36,13 +36,13 @@ namespace basic
   deallocHipDeviceData(x); \
   deallocHipDeviceData(y);
 
-__global__ void daxpy(Real_ptr y, Real_ptr x, 
-                      Real_type a, 
-                      Index_type iend) 
+__global__ void daxpy(Real_ptr y, Real_ptr x,
+                      Real_type a,
+                      Index_type iend)
 {
    Index_type i = blockIdx.x * blockDim.x + threadIdx.x;
    if (i < iend) {
-     DAXPY_BODY; 
```

Comment:
With HIP I noticed examples will use the following notation for threads/blocks ``hipThreadIdx_x`` or ``hipBlockIdx_x``. Do we know if there is a difference between them? 

See: 
https://rocmdocs.amd.com/en/latest/Programming_Guides/HIP-GUIDE.html

Reply:
I don't know, but apparently the cuda syntax works too.
We can try changing this in another PR if it matters. I guess we should check what syntax is used in RAJA as well while we're at it.